x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/action/SearchApplicationSearchRequest.java

```diff
@@ -65,6 +54,24 @@ public SearchApplicationSearchRequest(String name, @Nullable Map<String, Object>
         this.queryParams = Objects.requireNonNullElse(queryParams, Map.of());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Is it necessary to have `@SuppressWarnings` annotations both at the top level as well as inside the method?

Reply:
I agree, it isn't necessary. removed them in af753f213ae0513cd019ad628cadbedc1ab7ce08